scripts/docker_cmake

```diff
@@ -32,6 +32,7 @@ ARGS=(
     ### PACKAGES CONFIGURATION ###
     -D Trilinos_ENABLE_ALL_PACKAGES=OFF
     -D Trilinos_ENABLE_ALL_OPTIONAL_PACKAGES=OFF
+    "${Trilinos_ENABLE_Fortran}"
```

Comment:
I did this so the behavior doesn't change when using `gcc`. If I do what you propose, we will need a `docker_gcc_env.sh` to set `ENABLE_FORTRAN`.